source/administration/server-side-encryption/server-side-encryption-sse-kms.rst

```diff
@@ -95,20 +87,15 @@ This procedure requires the following components:
   instructions on downloading and installing ``mc``.
 
 
-- Install :minio-git:`MinIO Key Encryption Service (KES) <kes>` on a machine
-  with internet access. See the ``kes``
-  :minio-git:`Getting Started <kes/wiki/Getting-Started>` guide for 
-  instructions on downloading, installing, and configuring KES.
+- Install :kes-docs:`MinIO Key Encryption Service (KES) <>` on a machine with internet access. 
+  See the ``kes`` :kes-docs:`Getting Started <tutorials/getting-started/>` guide for instructions on downloading, installing, and configuring KES.
 
 1) Create an Encryption Key for SSE-KMS Encryption
```

Comment:
Note that we do need to flop over to API keys at some point.

Reply:
Added #1280 to track that needed work.